plugins/rabbitmq/pyparts/synchronization_checks.py

```diff
@@ -0,0 +1,30 @@
+from core.checks import CallbackHelper
+from core.issues import (
+    issue_types,
+    issue_utils,
+)
+from core.plugins.rabbitmq import RabbitMQEventChecksBase
+
+YAML_PRIORITY = 1
+EVENTCALLBACKS = CallbackHelper()
+
+
+class RabbitMQSynchronizationChecks(RabbitMQEventChecksBase):
+
+    def __init__(self):
+        super().__init__(yaml_defs_group='synced-queues',
+                         callback_helper=EVENTCALLBACKS)
+
+    @EVENTCALLBACKS.callback
+    def no_sync(self, event):  # pylint: disable=W0613
+        msg = ("Transient mirrored classic queues are not deleted when there "
+               "are no replicas available for promotion. Please stop all "
+               "rabbitmq-server units and restart them.")
```

Comment:
Should we perhaps be clearer about that the fact that it is necessary to stop all units before starting any back up again i.e. a rolling restart will not work here.